### ISSUE TITLE
Lower performance on customer dataset

### ISSUE THREAD

**Issue by Cli98** [zylo117/Yet-Another-EfficientDet-Pytorch#160]
Hi @zylo117 

Wish you a great day. I run your framework (v4) and it converges.  I get ap of ~23.6 on customer dataset with 10 categories. However, on mmdetection based fasterrcnn-fpn framework, I'm able to get 25.9 AP on 10 categories.

According to paper, v4 should be able to beat fasterrcnn by a large margin (~10 ap on coco 2017 test). I expected to see at least a higher ap on my custom dataset. But I'm not able to see it.

Apart from learning rate tuning and anchor setup, what else can you come up with? In the meanwhile I observed issues of NMS similar as #153 . Any idea on that?  

Thank you,

**Comment by zylo117** [comment]
He/She set `use_float16 = True`, so performance is different.
Did you train with `--head_only True`?
You should also check your loss graph to see if you were suffering overfitting.

**Comment by Cli98** [comment]
@zylo117  No I did not train with head. I observe no overfitting issues. Even if I do, that will not differ than 10 ap. Do you have any suggestions to improve performance? And do you think if performance of V4 can beat FasterRcnn-FPN?

**Comment by zylo117** [comment]
Yes D4 should be better than fasterrcnn, you can try training with `--optim sgd`

**Comment by Cli98** [comment]
Any suggestions for learning rate? Same as default?

**Comment by zylo117** [comment]
try 10 times smaller

**Comment by Cli98** [comment]
> try 10 times smaller

Cool. I will try this.
